_plugins/server_generator.rb

```diff
@@ -1,5 +1,7 @@
 module Jekyll
   class ServerPage < Page
+    include Jekyll::ThanksToFilter
```

Comment:
Yes, it's used in (and was originally written for) the thanks page, where it's used as a filter. It seems that the filters require it to be written as an instance method, so this was the easiest way to get it to work. Where's library code supposed to go in Jekyll? Is there a `_lib/` dir?